src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -705,36 +733,36 @@ public void testKeyPairFileDataSourceSerialization() throws Exception {
           String.format("alter user %s set rsa_public_key='%s'", params.get("user"), pubKey));
     }
 
-    Connection con = ds.getConnection();
-    ResultSet resultSet = con.createStatement().executeQuery("select 1");
-    resultSet.next();
-    assertThat("select 1", resultSet.getInt(1), equalTo(1));
-    con.close();
+    try (Connection con = ds.getConnection();
+        Statement statement = con.createStatement();
+        ResultSet resultSet = con.createStatement().executeQuery("select 1")) {
```

Comment:
Why not use statement.executeQuery("select 1")?

Reply:
I missed them. I change all the con.createStatement().esecuteQuery() to statement.createStatement in this file.